composer.json

```diff
@@ -27,28 +27,37 @@
     "config": {
         "sort-packages": true
     },
+    "repositories": [
+        {
+            "type": "git",
+            "url": "https://github.com/webimpress/zend-expressive-router.git"
+        },
+        {
+            "type": "git",
+            "url": "https://github.com/weierophinney/zend-stratigility.git"
+        }
+    ],
     "require": {
-        "php": "^5.6 || ^7.0",
+        "php": "^7.1",
         "fig/http-message-util": "^1.1.2",
-        "http-interop/http-middleware": "^0.4.1",
+        "http-interop/http-server-middleware": "^1.0.1",
```

Comment:
Should we explicity require `http-interop/http-server-handler` as well?

Reply:
No, because http-server-middleware already requires it (as the interface it defines is type-hinted against within `MiddlewareInterface`).